_redirects.yml

```
@@ -550,3 +550,7 @@
 "https://hub.docker.com/_/zookeeper/":
   - /samples/zookeeper
   - /samples/library/zookeeper
+
+# feedback
+"https://docs.google.com/forms/d/e/1FAIpQLSenjK8KiBiOrwCpq06u_iVEZGv4tfTEpfmY_cQDmsqJrHL9Gw/viewform":
+  - /feedback/extension/
```

I see it's missing a newline; not sure if this would cause issues, but perhaps better safe than sorry for that one 😅 

```suggestion
  - /feedback/extension/

```